src/StakingNodesManager.sol

```diff
@@ -119,6 +127,8 @@ contract StakingNodesManager is
 
     bool public validatorRegistrationPaused;
 
+    address public withdrawalAssetsVault;
```

Comment:
Please pay gray attention to memory layouts and that being respected when reviewed.